blog/integration-stack-4.3.0.Beta1-for-mars.adoc

```diff
@@ -0,0 +1,109 @@
+= Integration tooling available for Eclipse Mars.2
+:page-layout: blog
+:page-author: pleacu
+:page-date: 2015-03-11
```

Comment:
I was wondering for a few minutes why the page would not appear on the home page, until I realized that the year was wrong ;-)
I'll fix that by setting the publish date to today, then I'll publish it.